database/src/main/resources/db/migration/V1.87.0__RHCLOUD-29084_event_types_subscribed_by_default.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE event_type
+    ADD COLUMN subscribed_by_default BOOLEAN NOT NULL DEFAULT FALSE;
```

Comment:
I wonder if this column shouldn't be a text array of integration types.
I mean what if an application want to set Daily emails by default, but not Instant emails ?
WDYT ?

Reply:
If an application needs that in the future, yes we could use an array or a list here. Until then, I'd rather keep things simple and satisfy our current needs which are simpler than that.

Reply:
Ok then